.github/workflows/ci.yml

```
@@ -1,0 +1,468 @@
+      - test-go-api
+      - test-go-worker
+      - test-nextjs
+      - integration-tests
+      - validate-docker-build
+    steps:
+      - name: Check CI Status
+        run: |
+          echo "=
+ CI Pipeline Summary"
+          echo "====================="
+          echo "Lint Go: ${{ needs.lint-go.result }}"
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 428: mapping values are not allowed in this context

(syntax-check)